tensorflow/lite/micro/kernels/cmsis-nn/add.cc

```diff
@@ -13,9 +13,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
+#include "arm_nnfunctions.h"
```

Comment:
That wouldn't work when supplying an external CMSIS_PATH on command line.

When having
#include "arm_nnfunctions.h"
it can be found because we have
  INCLUDES += \
    -I$(CMSIS_PATH)/CMSIS/Core/Include \
    -I$(CMSIS_PATH)/CMSIS/DSP/Include \
    -I$(CMSIS_PATH)/CMSIS/NN/Include

When having 
#include "cmsis/CMSIS/NN/Include/arm_nnfunctions.h"
it can be found because we have this in Makefile
INCLUDES := \
-I. \
-I$(MAKEFILE_DIR)/downloads/ \

However with an external CMSIS_PATH there is no 'cmsis' in lower case in the path.

I propose to keep it as just
#include "arm_nnfunctions.h"
If that is OK?

Reply:
again, thinking some more, this might get in the way of Arduino.

Let me fix a few things for the Arduino before any updates on this PR.

Reply:
Ok